src/js/components/awardv2/idv/amounts/charts/ExceedsCurrentChart.jsx

```diff
@@ -6,8 +6,6 @@
 import React from 'react';
 import { generatePercentage } from 'helpers/aggregatedAmountsHelper';
 
-import InfoTooltip from 'components/awardv2/idv/InfoTooltip';
-import { awardAmountsOverspendingInfo } from 'components/awardv2/idv/InfoTooltipContent';
```

Comment:
Do we want to also remove `awardAmountsOverspendingInfo` from `components/awardv2/idv/InfoTooltipContent`?